src/pages/learn.tsx

```diff
@@ -18,10 +19,11 @@ export type Category = {
 }
 
 export const ArticlePreviewCard = (props: any) => {
-  const url = '/learn/' + props.articleData.slug; 
+  const localePrefix = props.locale ? ("/" + props.locale) : "";
+  const url = localePrefix + '/learn/' + props.articleData.slug; 
   const categoryLabels = (props.articleData.categories).map(
     (category: Category, i: number) => 
-    <Link key={i} to={'/learn/category/' + category.slug} className="tag is-primary is-light is-uppercase">
+    <Link key={i} to={localePrefix + '/learn/category/' + category.slug} className="tag is-primary is-light is-uppercase">
```

Comment:
adding this "localePrefix" to each internal link was definitely the piece of this PR that felt the least "DRY" to me... it definitely wasn't a huge lift, but I wonder if there is some kind of global "locale" prop we can define and access that would make this easier.

Reply:
Ah right, I had a similar issue in WoW's localization and my solution was to make [a `<LocaleLink>` component that auto-prefixes the path](https://github.com/JustFixNYC/who-owns-what/blob/ccf0e526f7e6a259c0d39d1e0d8f93a8fefbbb0e/client/src/i18n.tsx#L170-L181).  

A lot of it depends on where you're storing the information about what the current locale is, though; I think a [React context](https://reactjs.org/docs/context.html) might be the easiest approach, so you don't have to explicitly pass down the current locale through the whole component hierarchy.

Reply:
ooooh ok i'll take a look!

Reply:
I just added this as a clubhouse card in our backlog: https://app.clubhouse.io/justfixnyc/story/1653/convert-all-internal-links-to-localelinks-like-in-wow